src/main/scala/nl.knaw.dans.easy.dd2d/mapping/Format.scala

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.dd2d.mapping
+
+import nl.knaw.dans.easy.dd2d.dataverse.json.{ FieldMap, JsonObject }
+
+import scala.xml.Node
+
+object Format extends BlockContentTypeAndFileFormat {
+
+  val imtFormats = Map(
+    "application/postscript" -> "application/postscript",
```

Comment:
A bit weird that nothing is mapped to something else. Why not use a simple list here then?